src/Provider/DataSourceProvider.js

```diff
@@ -47,16 +51,26 @@ function FetchAndConvertSourceData(url, layer, extentSource, extentDestination)
         withAltitude: layer.isGeometryLayer,
     };
 
+    data.coords = extSrc;
+
+    return parser(data, options).then((parsedFile) => {
+        if (source.isFileSource) {
+            source.parsedData = parsedFile;
+            if (parsedFile.extent._crs != 'EPSG:4978') {
```

Comment:
We currently have some problems where the `parsedFile.extent` is in `4978`: this makes no sense (in itowns at least) to have a geocentric extent. This problem is not fixed in this PR, but an issue will be opened about it.

Reply:
could you open it?

Reply:
Done in #1063